tests/leds/Makefile

```diff
@@ -1,3 +1,5 @@
+# Some boards do not initialize LED0 by default
+CFLAGS=-DAUTO_INIT_LED0
```

Comment:
please move this change after `include ../Makefile.tests_common`

shouldn't make a difference but is structural (more) consistent with other Makefiles

otherwise, looks good and simple. To me no need to have some BOARDs list that would need update all the time.

Reply:
Ok. Moved the line of code.